README.md

```diff
@@ -1 +1,7 @@
-# Traffic
```

Comment:
Like that you guys have a readme, but would like to see more info about the projects goals and a guide on how to actually use the code, preferably with demos

Reply:
Excellent idea